org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/XtextUiModule.java

```diff
@@ -184,5 +186,8 @@ public void configureIResourceDescriptionsPersisted(com.google.inject.Binder bin
 	public void configureEditorScope(Binder binder) {
 		binder.bindConstant().annotatedWith(Names.named(XtextEditor.KEY_BINDING_SCOPE)).to("org.eclipse.xtext.xtext.ui.XtextEditorScope");
 	}
-	
+
```

Comment:
@tivervac why is this here and not in DefaultUiModule?

Reply:
Because I thought that was the place to put it, seems I was wrong ;)